devlib/utils/ssh.py

```diff
@@ -247,7 +249,9 @@ def background(self, command, stdout=subprocess.PIPE, stderr=subprocess.PIPE, as
             keyfile_string = '-i {}'.format(self.keyfile) if self.keyfile else ''
             if as_root and not self.connected_as_root:
                 command = self.sudo_cmd.format(command)
-            command = '{} {} {} {}@{} {}'.format(ssh, keyfile_string, port_string, self.username, self.host, command)
+            options = " ".join([ "-o {}={}".format(key,val) for key,val in self.options.items()])
+            command = '{} {} {} {} {}@{} {}'.format(ssh, options, keyfile_string, port_string, \
```

Comment:
Do not need `\` inside parens. Also could you please also break up the other lines that are too wide to fit in the GitHub review viewport without scrolling.